api/src/app.ts

```diff
@@ -2,11 +2,11 @@ import { handlePressure } from '@directus/pressure';
 import cookieParser from 'cookie-parser';
 import type { Request, RequestHandler, Response } from 'express';
 import express from 'express';
-import type { ServerResponse } from 'http';
+import type { ServerResponse } from 'node:http';
 import { merge } from 'lodash-es';
 import { readFile } from 'node:fs/promises';
 import { createRequire } from 'node:module';
-import path from 'path';
+import path from 'node:path';
```

Comment:
just out of curiosity: was the `node` prefix necessary for the new bundling process or just to be more expressive?

Reply:
😃 
<img width="813" src="https://github.com/directus/directus/assets/5363448/94bc7416-29fb-4d3e-bd8a-6c029dfcf6f6">

To expand on that:
* Nowadays, using the `node:` prefix to import Node.js built-in modules is the recommended way for various reasons
* The idea here is to preemptively ensure that  the bundle won't contain multiple import statement for the same module (`path` and `node:path`)

Reply:
Uups 😄👍 alright, thanks!!